components/benefit_sponsors/app/models/benefit_sponsors/services/upload_documents_to_profiles_service.rb

```diff
@@ -0,0 +1,79 @@
+module BenefitSponsors
```

Comment:
Class Name renamed to use this Service as common service for both invoices and commission statements.